routes/w3c/groups.json

```diff
@@ -1,74 +1,1876 @@
 {
   "wg": {
-    "audio": 46884,
```

Comment:
Lets not commit this file anymore? It can be fetched with `update-data-sources`.